### PR TITLE
retrive data from db and send it to FE

### LINKED ISSUE #14: Add `/contractor-results`
- Create a GET request route for `/contractor-results/:job` based on selected option
- Get contractor data from DB (ordered by the rating), and return to FE as JSON

### PR DESCRIPTION
**Pull request by ebraheemabbas**
adding new route to get data from db and send it back to front end as json
closes #14

### CODE REVIEW

File: the-experts-BE/app.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var logger = require('morgan');
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 var testAPIRouter = require("./routes/testAPI");
+var getContractor=require("./routes/contractor-results");
```
Comment (jamiecoe) [inline the-experts-BE/app.js]: Why are you using `var` for all these variables? You are not mutating their values, you should be using `const`.

File: the-experts-BE/routes/contractor-results.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var express = require("express");
```
Comment (jamiecoe) [inline the-experts-BE/routes/contractor-results.js]: Again, you should be using `const` here as the variables aren't being mutated

File: the-experts-BE/routes/contractor-results.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+var router = express.Router();
+var queries= require("../queries/index")
+router.get("/", function(req, res, next) {
+    console.log("hellow world")
```
Comment (jamiecoe) [inline the-experts-BE/routes/contractor-results.js]: Please remove random `console.log()` from production code

File: the-experts-BE/routes/contractor-results.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    queries.getContractor(req.params.job,(err,response)=>{
+        if(err) console.log(err);
```
Comment (jamiecoe) [inline the-experts-BE/routes/contractor-results.js]: Why are you just `console.log()` this error? You have error handling middleware setup, use `next()` to pass the error to it

**Review by jamiecoe — CHANGES_REQUESTED** [review]

**Review by jamiecoe — APPROVED** [review]

**Review by omriza5 — APPROVED** [review]

**Review by HosenAgbaria — APPROVED** [review]

**Pull request merged**